packages/deployments/contracts/contracts/core/connext/facets/BridgeFacet.sol

```diff
@@ -533,10 +538,12 @@ contract BridgeFacet is BaseConnextFacet {
       _params.nonce = s.nonce++;
     }
 
-    // Store the relayer fee.
+    // Handle the relayer fee.
```

Comment:
Remove `NOTE` below: I think it's not relevant anymore, as it was referring to handling native assets as input assets

Reply:
note is still relevant for event purposes -- basically just saying that the bridged amount is what is hashed in the transfer id so you need to wait until after swap to make sure you have the right transfer